packages/app/src/youtube.js

```diff
@@ -1,12 +1,13 @@
 import puppeteer from 'puppeteer';
-const YOUTUBE_BASE_URL = 'https://soundcloud.com/discover';
+import { SSL_OP_EPHEMERAL_RSA } from 'constants';
+const YOUTUBE_BASE_URL = 'https://www.youtube.com';
 const youtubeSelectors = {
-  searchField: ".headerSearch__input",
-  searchListItem: ".sound__body",
-  itemName: ".sc-link-dark span",
-  itemUrl: ".sc-link-dark",
-  itemUser: ".soundTitle__usernameText",
-  itemImage: "",
+  searchField: "#search",
+  searchListItem: "#dismissable",
```

Comment:
If you check the results it includes a lot of wrong results so the selector for searchlistitem is not good.

Reply:
Check document.querySelectorAll("ytd-video-renderer")